lib/Github/Api/Repository/Releases.php

```diff
@@ -68,6 +68,26 @@ public function show($username, $repository, $id)
         return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/releases/'.$id);
     }
 
+    /**
+     * Generate release notes content for a release.
+     *
+     * @param string $username
+     * @param string $repository
+     * @param array  $params
+     *
+     * @throws MissingArgumentException
+     *
+     * @return array
+     */
+    public function generateNotes($username, $repository, array $params)
+    {
+        if (!isset($params['tag_name'])) {
+            throw new MissingArgumentException('tag_name');
+        }
```

Comment:
Let's remove this check and fully rely on github to do the validation, if they change anything to the required parameters in the future we don't need a change in the library to be compatible.

Reply:
Fair. I was just following what `create()` does in the same class but happy to remove this validation.